scene/gui/text_edit.h

```diff
@@ -632,6 +633,10 @@ class TextEdit : public Control {
 	int get_indent_level(int p_line) const;
 	bool is_line_comment(int p_line) const;
 
+	bool is_line_empty_or_comment(int p_line) const;
+	void set_debug_comment_hide_cursor(bool hide, bool default_highlite);
```

Comment:
`debug_comment_hide_cursor` is not exposed. Also typo in `default_highlite` (→ `default_highlight`). And parameters should start with `p_`.

I'm pretty sure this function should not be here, but instead `CodeTextEditor` (or some other class) can call `set_highlight_current_line`. If the `_toggle_draw_caret` call is wanted, maybe it can be unfocused, or, failing that, `set_color("caret_color", ...)` might be used.
If it is needed, maybe it should be renamed to `set_cursor_visible` and `is_cursor_visible`, and the `default_highlight` parameter dropped.

Reply:
done